src/database_metrics/operations.py

```diff
@@ -0,0 +1,75 @@
+"""Operations for which metrics are captured for."""
+import requests
+from requests import Response
+
+from docker import DockerClient
+
+
+def send_query(params: dict) -> Response:
+    """
+    Send query to Epidata API running on docker MariaDB container.
+
+    Parameters
+    ----------
+    params: dict
+        Query parameters to send. List of parameters can be found here:
+        https://cmu-delphi.github.io/delphi-epidata/api/covidcast.html#constructing-api-queries
+
+    Returns
+    -------
+    Requests Response object.
+    """
+    req = requests.get("http://localhost:10080/epidata/api.php",
+                       auth=("user", "pass"),
```

Comment:
what are these creds used for? the api shouldn't require http auth.

Reply:
oops didn't even realize this, was using them for some other queries and just threw them in. removed.